reference/resources/hostendpoint.md

```diff
@@ -38,6 +33,11 @@ aliases are supported (all case insensitive): `hostendpoint`, `hostendpoints`, `
 > is omitted then security rules that use labels will fail to match this endpoint.
 {: .alert .alert-danger}
 
+{{site.prodname}} provides a default profile named `projectcalico-allow-all` that may be added to a host endpoint to change the host endpoint's default behavior. This profile consists of ingress and egress rules that allow all traffic. When the `projectcalico-allow-all` profile is attached to a host endpoint, the default behavior of that host endpoint becomes an allow-all.
```

Comment:
It's a shame we didn't call this default allow rather than allow all... but probably too much work to change it now.

Reply:
Actually the PR for this profile resource is under review. I'll make the naming change and see what the reviewers think.